docs/dev/setup/programming-exercises.rst

```diff
@@ -0,0 +1,57 @@
+Adjustments for programming exercises
```

Comment:
GitHub Actions complains: `docs/dev/setup/programming-exercises.rst#L1: document isn't included in any toctree`

Reply:
Should be fixed now.